test/csuite/Makefile.am

```diff
@@ -7,6 +7,10 @@ AM_LDFLAGS = -static
 all_TESTS=
 noinst_PROGRAMS=
 
+test_wt7989_compact_checkpoint_SOURCES = wt7989_compact_checkpoint/main.c
```

Comment:
This should be moved below too to keep the order

Reply:
Done, thanks!